dlang.org.ddoc

```diff
@@ -493,7 +493,9 @@ $(SUBNAV_TEMPLATE
         $(ROOT_DIR)articles/template-comparison.html, Template Comparison,
         $(ROOT_DIR)articles/d-array-article.html, D Slices,
         $(ROOT_DIR)articles/cppcontracts.html, D's Contract Programming,
-        $(ROOT_DIR)articles/dll-linux.html, Writing Shared Libraries
+        $(ROOT_DIR)articles/dll-linux.html, Writing Shared Libraries for Linux,
+        $(ROOT_DIR)articles/RefReturnScope.html, Coralling Wild Pointers With $(D ref return scope),
```

Comment:
Please try to remove special formatting from the navigation, I don't think we can use it in all circumstances.

Reply:
that did the trick, thanks!